pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java

```diff
@@ -75,4 +90,29 @@ public WorkflowResponse buildStateResponse(Handle handle, String userGuid, Workf
         }
         return WorkflowResponse.unknown();
     }
+
+    private void createActivityInstanceIfMissing(
+            Handle handle,
+            WorkflowState fromState,
+            WorkflowState nextState,
+            String operatorGuid,
+            String userGuid
+    ) {
+        if (nextState.getType() != StateType.ACTIVITY) {
+            return;
+        }
+        ActivityState activityState = (ActivityState) nextState;
+        String instanceGuid = handle.attach(JdbiActivityInstance.class)
+                .findLatestInstanceGuidByUserGuidAndActivityId(userGuid, activityState.getActivityId())
+                .orElse(null);
+        boolean activityInstanceIsMissing = instanceGuid == null;
+        if (activityInstanceIsMissing) {
+            ActivityInstanceDto instanceDto = handle.attach(ActivityInstanceDao.class)
+                    .insertInstance(activityState.getActivityId(), operatorGuid, userGuid, InstanceStatusType.CREATED, false);
+            LOG.info("Created start activity instance with guid '{}' for user guid '{}' using operator guid '{}' and activity id {}",
```

Comment:
```suggestion
            LOG.info("Created activity instance with guid '{}' for user guid '{}' using operator guid '{}' and activity id {}",
```

Reply:
Since it's no longer just the "start" activity, we should remove `start` from this log.

Reply:
Done.